var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -250,6 +243,7 @@ class Trilinos(CMakePackage):
     conflicts('+belos', when='~teuchos')
     conflicts('+epetraext', when='~epetra')
     conflicts('+epetraext', when='~teuchos')
+    conflicts('+exodus', when='~netcdf')
```

Comment:
Not really part of this PR.

Reply:
Separate commit, though? Could rebase and merge. Found while testing fortrilinos.

Reply:
This is fine.